File: 백준/Silver/2839. 설탕 배달/설탕 배달.py
sugar = int(input())
cnt=0
if sugar%5==0:
    print(sugar//5)
else:
    while True:
        sugar-=3
        cnt+=1
        if sugar%5==0:
            cnt+=sugar//5
            print(cnt)
            break
        if sugar<0:
            print(-1)
            break

File: 백준/Silver/2839. 설탕 배달/설탕 배달.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int sugar, cnt=0;
        Scanner scanner = new Scanner(System.in);
        sugar = scanner.nextInt();
        if (sugar%5==0)
            System.out.println(sugar/5);
        else {
            while (true) {
                sugar-=3;
                cnt++;
                if (sugar%5==0) {
                    cnt+=sugar/5;
                    System.out.println(cnt);
                    break;
                }
                if (sugar==0)
                    System.out.println(cnt);
                if (sugar<0) {
                    System.out.println(-1);
                    break;
                }
            }
        }
    }
}